app/controller/auth.js

```diff
@@ -1,9 +1,12 @@
+// Imports for packages
 import passport from 'passport';
 import { Strategy as GitHubStrategy } from 'passport-github';
 
+// Local Imports
```

Comment:
ah, i got you. I'm actually a really big fan of that ordering for imports — in pretty much every language I use I tend to organize my imports into three different sections (sometimes separated by a newline): standard/system imports, third party imports, first party/local imports.

So, my $0.02 is to keep the line breaks to reinforce the separation between those three logical groups, but ditch the comments because it's annoying to type them. But like I've said, this is very much a personal preference and if you feel really strongly, I don't want to stand in your way